Command/DoctrineCommand.php

```diff
@@ -45,7 +45,7 @@ public static function configureMigrations(ContainerInterface $container, Config
             $configuration->setMigrationsDirectory($dir);
         } else {
             // class Kernel has method getKernelParameters with some of the important path parameters
-            $pathPlaceholderArray = ['kernel.root_dir', 'kernel.cache_dir', 'kernel.logs_dir'];
+            $pathPlaceholderArray = ['kernel.project_dir', 'kernel.cache_dir', 'kernel.logs_dir'];
```

Comment:
I think this can be a BC break? The code below seems to replace/resolve parameter placeholders with kernel parameters. So now this will not work anymore if my Symfony 3.4 project relies on `kernel.root_dir`?

Reply:
So I think for now we can just add `kernel.project_dir` to the array but also keep `kernel.root_dir`. There is a check below that ensures the kernel parameter exists.